### PR TITLE
Fixed #4128 - Broken link about commitment during covid-19

### LINKED ISSUE #4128: " Our commitment to customers during COVID-19" link does not work
" Our commitment to customers during COVID-19" link does not work

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4faeeb9b-7cf7-64db-1c0b-17fc4a21f422
* Version Independent ID: ee0660a9-3902-5ce2-c9ea-82e23bd99dcd
* Content: [FAQ - Support your remote workforce - Microsoft Teams](https://docs.microsoft.com/en-us/MicrosoftTeams/faq-support-remote-workforce#feedback)
* Content Source: [Teams/FAQ-support-remote-workforce.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/FAQ-support-remote-workforce.md)
* Service: **msteams**
* GitHub Login: @LolaJacobsen
* Microsoft Alias: **lolaj**

### PR DESCRIPTION
**Pull request by dotwhat**
Fixed broken link of 'Our commitment to customers during COVID-19' from: 
https://www.microsoft.com/microsoft-365/blog/2020/03/05/our-commitment-to-customers-during-covid-19/
To: 
https://www.microsoft.com/en-us/microsoft-365/blog/2020/03/05/our-commitment-to-customers-during-covid-19/
(it was missing the language code in the url)

### CODE REVIEW

**Comment by dotwhat** [comment]
#4128

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/FAQ-support-remote-workforce.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Microsoft has been significantly expanding capacity in key regions with the rece
 > [!NOTE]
 > If you're already set up with M365, and want to know more about the actions Microsoft is taking, please go to the Message Center and review the **MC205458** message center post.
 
-Don't miss our March 5, 2020 blog post from Jared Spataro, Corporate Vice President for Microsoft 365: [Our commitment to customers during COVID-19](https://www.microsoft.com/microsoft-365/blog/2020/03/05/our-commitment-to-customers-during-covid-19/)
+Don't miss our March 5, 2020 blog post from Jared Spataro, Corporate Vice President for Microsoft 365: [Our commitment to customers during COVID-19](https://www.microsoft.com/en-us/microsoft-365/blog/2020/03/05/our-commitment-to-customers-during-covid-19/)
```
Comment (JohanFreelancer9) [inline Teams/FAQ-support-remote-workforce.md]:
```suggestion
Don't miss our March 5, 2020 blog post from Jared Spataro, Corporate Vice President for Microsoft 365: [Our commitment to customers during COVID-19](https://www.microsoft.com/en-us/microsoft-365/blog/2020/03/05/our-commitment-to-customers-during-covid-19/).
```

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**